src/ppx/output_module.ml

```diff
@@ -1024,13 +1026,9 @@ let generate_fragment_implementation config name
       graphql_external config (Graphql_ast.Fragment fragment);
       [
         [%stri let query = [%e printed_query]];
+        wrap_as_uncurried_fn [%stri let parse (value : Raw.t) = [%e parse_fn]];
         wrap_as_uncurried_fn
-          [%stri
-            let parse = (fun value -> [%e parse_fn] : Raw.t -> [%t type_name])];
-        wrap_as_uncurried_fn
-          [%stri
-            let serialize =
-              (fun value -> [%e serialize_fn] : [%t type_name] -> Raw.t)];
```

Comment:
I've fiddled with this a few times and all the options seem to generate a lot of changes in the printed output.

`generate_operation_implementation` doesn't include any type annotations here, would that be preferable for this as well?